src/libYARP_OS/include/yarp/os/Random.h

```diff
@@ -26,7 +28,9 @@ namespace yarp {
  *
  * A collection of basic random number generation algorithms.
  */
-class YARP_OS_API yarp::os::Random {
+class YARP_OS_API YARP_OS_DEPRECATED_API YARP_OS_DEPRECATED_API_MSG("use instead utilities from C++11's <random> lib") yarp::os::Random {
```

Comment:
`YARP_OS_DEPRECATED_API` and `YARP_OS_DEPRECATED_API_MSG` are not supposed to be used together, the first is used to deprecate with a standard message, the second with a custom message.
please remove `YARP_OS_DEPRECATED_API`

Reply:
:+1: , https://github.com/robotology/yarp/pull/1418/commits/dac3e586fc372f5174712721f0968cfa472b3da3